README.md

```diff
@@ -2,17 +2,12 @@
 
 The Glutton server listens on both TCP and UDP port 5000 for new connections.
 
-First make sure you have installed iptables-persistent. During installation select YES for saving your current firewall rules for both ipv4 and ipv6.
+Install required packages -
 ```
-apt-get install iptables-persistent conntrack golang libpcap-dev
-```
-Download and set up glutton, add GOPATH to /etc/environment. Example
-```
-mkdir /opt/go
-echo export GOPATH=/opt/go >> /etc/environment
-source /etc/environment
-go get github.com/mushorg/glutton
+apt-get install iptables-persistent conntrack libpcap-dev libnetfilter-queue-dev golang
```

Comment:
This is not correct anymore. Here are the packages required: go 1.7+, libnetfilter-queue-dev, libpcap-dev, iptables-dev